lxd/api.go

```diff
@@ -106,7 +106,8 @@ func restServer(d *Daemon) *http.Server {
 		uiHandlerErrorUINotEnabled := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			w.Header().Set("Content-Type", "text/html")
 			w.WriteHeader(http.StatusServiceUnavailable)
-			fmt.Fprint(w, errorMessage)
+			_, err := fmt.Fprint(w, errorMessage)
+			logger.Warn(fmt.Sprintf("Could not write error message to writer object: %v", err), logger.Ctx{"url": r.URL, "method": r.Method, "remote": r.RemoteAddr})
```

Comment:
Mixing contextual and non-contextual logging not advisable.
Suggest adding an `err` field to the context for the `err` value, this is what we normally do.